provider/resources.go

```diff
@@ -2037,6 +2038,7 @@ func Provider() tfbridge.ProviderInfo {
 			"aws_db_proxy_default_target_group": {Tok: awsResource(rdsMod, "ProxyDefaultTargetGroup")},
 			"aws_db_proxy_target":               {Tok: awsResource(rdsMod, "ProxyTarget")},
 			"aws_db_proxy_endpoint":             {Tok: awsResource(rdsMod, "ProxyEndpoint")},
+			"aws_rds_cluster_role_association":  {Tok: awsResource(rdsMod, "RdsClusterRoleAssociation")},
```

Comment:
Should it be `ClusterRoleAssociation`? `aws_rds_*` resources seem to omit that prefix. Also, consider moving the new resource next to other `aws_rds_*` resources above.

Reply:
👌